docs/self-managed/operational-guides/backup-restore/webapps-backup.md

```diff
@@ -255,11 +315,7 @@ curl --request POST `http://localhost:9200/_snapshot/test/camunda_operate_123_8.
 
 To summarize, the process may look as follows:
 
-1. Stop Operate.
-2. Ensure there are no Operate indices present in Elasticsearch (otherwise the restore process will fail).
+1. Stop all webapps.
+2. Ensure there are no webapp indices present in Elasticsearch (otherwise the restore process will fail).
 3. Iterate over all Elasticsearch snapshots included in the desired backup and restore them using the Elasticsearch restore snapshot API.
-4. Start Operate.
-
-## Backup and restore of Tasklist data
-
-Backup and restore of Tasklist may be performed in exactly the same way as [Operate data](#).
+4. Start all webapps.
```

Comment:
❓ Should we note that Zeebe also need to be restored? Because webapp independently from Zeebe restoring will not really work right?